lua_examples/net_send_queue.lua

```diff
@@ -19,12 +19,11 @@
 --
 ------------------------------------------------------------------------------
 do
-  -- cache
-  local shift = table.remove
   -- queue handler
   local make_sender = function(socket)
-    local queue = { }
-    local is_sending = false
+    local shift = table.remove
+    local max_send, queue, is_sending = 1460, { }, false
```

Comment:
I'm not deeply enough involved in networking these days to remember where 1460 comes from, I had to look it up. So, personally I'd welcome if a comment like below would clarify that:
```lua
-- 1460 is the standard Ethernet frame payload size in bytes (1500 - IP header - TCP header).
-- See https://en.wikipedia.org/wiki/Jumbo_frame#Bandwidth_efficiency for details.
```